docs/upgrading/upgrading_to_v0x.md

```diff
@@ -24,6 +24,10 @@ We decided to move away from [Hungarian notation](https://en.wikipedia.org/wiki/
 - `BaseBrowserController` -> `BrowserController`
 - `BaseBrowserPlugin` -> `BrowserPlugin`
 
+### EnqueueStrategy
+
+The `EnqueueStrategy` has been changed from an enum to a string literal type. All its values and their meaning remain unchanged.
```

Comment:
Did you consider making this change backwards compatible and accepting both the enum and the plain string?

Reply:
Is it really necessary, considering we're still 0.X?

Reply:
unpleasant but tolerable, especially when they are well documented ...

so you would rather prefer to keep both enum & literal types?

Reply:
I don't know. The advantage is avoiding a BC break, disadvantages are a longer type (more confusing documentation) and more code to handle it. Am I missing anything?

Reply:
You're right. The disadvantage is a BC break, that's clear from the beginning, and I accepted it going in. Having both options is IMO the worst case. I'd rather stick with enum than have both. This PR should be about correcting past mistakes, but not by creating a new mess.

Reply:
I'm not sure if I'd call using an enum a mistake :slightly_smiling_face: but this is more consistent and probably more friendly to the end user, so let's do it and never look back.